graphene/tests/issues/test_870.py

```diff
@@ -0,0 +1,80 @@
+import graphene
+
+
+class SecondChild(graphene.ObjectType):
+    hello = graphene.String()
+
+
+def resolve_second_child(self, info):
+    return SecondChild(hello=self.second_child._test)
+
+
+second_child_f = graphene.Field(SecondChild, resolver=resolve_second_child)
```

Comment:
Why do you define fields separately and not inside the class?
```
class FirstChild(graphene.ObjectType):
     hello = graphene.String()
     second_child = graphene.Field(SecondChild)

     def resolve_second_child(self, info):
        return SecondChild(hello=self.second_child._test)
```

Reply:
I felt it made the steps more explicit, but if that's the preference/code style for the test, I can make the change